connector/src/main/java/org/geysermc/connector/command/defaults/VersionCommand.java

```diff
@@ -62,28 +67,31 @@ public void execute(GeyserSession session, CommandSender sender, String[] args)
         sender.sendMessage(LanguageUtils.getPlayerLocaleString("geyser.commands.version.version", sender.getLocale(), GeyserConnector.NAME, GeyserConnector.VERSION, GeyserConnector.MINECRAFT_VERSION, bedrockVersions));
 
         // Disable update checking in dev mode
+        // Disable update checking for players in Geyser Standalone
         //noinspection ConstantConditions - changes in production
         if (!GeyserConnector.VERSION.equals("DEV")) {
-            sender.sendMessage(LanguageUtils.getPlayerLocaleString("geyser.commands.version.checking", sender.getLocale()));
-            try {
-                Properties gitProp = new Properties();
-                gitProp.load(FileUtils.getResource("git.properties"));
+            if (!(!sender.isConsole() && connector.getPlatformType() == PlatformType.STANDALONE)) {
```

Comment:
You can add this to the above `if` statement check.

Reply:
Will do, wasn't sure if it was getting too messy